kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2744 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    s3_path = f"s3://nex-gddp-cmip6/NEX-GDDP-CMIP6/{model}/historical/r1i1p1*/{variable}/*"
</pre><p>mention what <code>tasmax</code> is. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>